aws/artifacts/outputs.tf

```
@@ -0,0 +1,13 @@
+output "access_key_id" {
+  value       = "${module.user.access_key_id}"
```

```suggestion
  value       = "${join("", module.user.*.access_key_id})"
```

since it has `count`

---

Don't need to use this pattern with modules.